### PR TITLE
ui refactoring  : part 1 - procedures

### PR DESCRIPTION
**Pull request by naveen17797**
@teryhill  this is progressing phase of the ui refactoring
team have a look at this @2208Abhinav  @apooravc  and make the changes
Make sure you include the headers.inc.php which contains the color picker controller.

### CODE REVIEW

**Comment by 2208Abhinav** [comment]
@naveen17797 working super fast. :nerd_face:

**Comment by naveen17797** [comment]
RESULTS OF THIS PR ARE BELOW
procedures section completed.

screenshot from 2018-04-04 20-39-34
screenshot from 2018-04-04 20-39-18
screenshot from 2018-04-04 20-38-01
screenshot from 2018-04-04 20-37-45
screenshot from 2018-04-04 20-37-30

File: interface/themes/style.php [hunk 1]
```diff
@@ -70,15 +70,37 @@ function validate_html_color_code($arg) {
...
   background-color: #$secondary_color;
   color: #$secondary_font_color;
 }
 
-input[type='submit'], input[type='button'],input[type='submit'], button, a[role='button'] {
-	background-color: #$button_color;
-	color: #$button_font_color;
-	border: 2px solid #$button_font_color;
-}";
+.cp-positive {
```
Comment (teryhill) [inline interface/themes/style.php]: @aethelwulffe  these are temporary until the ui for changing them gets done.

Comment (aethelwulffe) [reply interface/themes/style.php]: OK.  Got it.

**Comment by teryhill** [comment]
I did not see any issues. I will merge after Art's input

**Pull request merged**